internal/backends/sqlite/metadata/settings.go

```diff
@@ -26,6 +26,13 @@ import (
 // Settings represents collection settings.
 type Settings struct {
 	Indexes []IndexInfo `json:"indexes"`
+	Capped  *Capped     `json:"capped,omitempty"` // nil if not capped
```

Comment:
Again, I think a flat structure will be easier to use. It removes cases like `Capped: &Capped{Size: 0}` – not nil, but zero